src/code42cli/cmds/auditlogs.py

```diff
@@ -103,11 +124,18 @@ def search(
     affected_user_id,
     affected_username,
     format,
+    use_checkpoint,
```

Comment:
This is not a good name for this... This describes a boolean.

I know we probably can't change this now to maintain consistency with the other checkpoints, but we could at least do something like this:

```
if use_checkpoint:
  checkpoint_name = use_checkpoint
```

Seems redundant but would improve readability, as I just wasted like 2 minutes wondering why we have to pass a boolean into the `_dedup...()` method

Reply:
You're right, I've updated to make it clear.